sway-lib-std/src/vec.sw

```diff
@@ -14,7 +14,7 @@ struct RawVec<T> {
 impl<T> RawVec<T> {
     /// Create a new `RawVec` with zero capacity.
     fn new() -> Self {
-        Self {
+        RawVec {
```

Comment:
Are these changes required?  I'm surprised if it used to work but no longer does.  Or is it more about verbosity and self-documentation?

Reply:
It used to be `RawVec` and was changed to `Self` recently by Emily because she introduced a change that allows the use of `Self` in this context. I suggest we keep it as `Self`.

Reply:
Reverted back, as it works now